authorization/v1/generated.pb.go

```diff
@@ -1110,64 +1110,64 @@ var fileDescriptor_39b89822f939ca46 = []byte{
+	0xdd, 0xd3, 0x19, 0x16, 0x51, 0xf9, 0x60, 0x40, 0x8d, 0x70, 0x40, 0xe0, 0x5f, 0x44, 0x08, 0x50,
+	0x27, 0x08, 0x36, 0xd3, 0x49, 0x2e, 0x20, 0x7e, 0x47, 0xf1, 0xf8, 0xb5, 0xce, 0x6c, 0xe6, 0x9c,
+	0x50, 0xfe, 0x3e, 0x3f, 0xcf, 0x48, 0xee, 0x09, 0xfc, 0x08, 0xaa, 0xbc, 0xa4, 0xdd, 0xf0, 0x58,
+	0xda, 0x7a, 0x3b, 0x83, 0x42, 0x62, 0xf6, 0x8f, 0x68, 0x22, 0xde, 0x5d, 0x12, 0x87, 0x24, 0x29,
+	0x07, 0x7f, 0x1f, 0xd6, 0x45, 0x91, 0x47, 0x65, 0xfb, 0x31, 0x7f, 0x23, 0x83, 0xec, 0xe4, 0x82,
+	0xd8, 0xde, 0x98, 0x8c, 0xeb, 0x33, 0x6b, 0x23, 0x99, 0x11, 0x85, 0x7f, 0x89, 0xe0, 0x1a, 0xa3,
+	0xee, 0xc8, 0x34, 0xa8, 0x6e, 0x18, 0xce, 0xd0, 0xf6, 0xa2, 0x8a, 0xf8, 0xfd, 0xec, 0xad, 0x0c,
+	0x8a, 0x1c, 0xf8, 0xbc, 0x5a, 0x3e, 0xaf, 0xa8, 0x46, 0xd7, 0x27, 0xe3, 0xfa, 0xb5, 0xb9, 0x60,
+	0x32, 0x5f, 0x0b, 0xf5, 0x8f, 0x08, 0x4a, 0x17, 0xb4, 0xc9, 0x7f, 0x35, 0x9e, 0x8f, 0x99, 0x07,
+	0xf7, 0xf4, 0xec, 0xfb, 0x0f, 0x82, 0x2b, 0x07, 0xd4, 0x3a, 0x92, 0x2d, 0xd8, 0xbf, 0x19, 0xfd,
+	0x91, 0x28, 0x28, 0x73, 0x94, 0xb9, 0xcc, 0xd3, 0x19, 0xce, 0x2b, 0x73, 0x7c, 0x0c, 0x45, 0xe6,
+	0xe9, 0xde, 0x30, 0xb8, 0x0c, 0xef, 0x64, 0x11, 0x35, 0x2b, 0x46, 0xb0, 0x6a, 0xaf, 0x49, 0x41,
+	0x45, 0xff, 0x9b, 0x48, 0x11, 0xea, 0xf7, 0x60, 0x73, 0xbe, 0x7a, 0x91, 0x85, 0x17, 0x9d, 0xc7,
+	0xc2, 0x6b, 0xc1, 0xd5, 0x64, 0x9a, 0xc9, 0xab, 0x6b, 0x81, 0x75, 0x29, 0x36, 0x30, 0xe6, 0x4e,
+	0x1e, 0x18, 0xd5, 0xbf, 0x20, 0x98, 0x9f, 0xd5, 0xf8, 0x47, 0x08, 0xaa, 0xf1, 0xc4, 0xf6, 0x37,
+	0x92, 0xca, 0x4e, 0xfb, 0x0c, 0x45, 0x15, 0xdc, 0xc4, 0xd3, 0x29, 0x32, 0x8e, 0xc0, 0x48, 0x52,
+	0x26, 0xd6, 0x00, 0xa6, 0x2a, 0xc7, 0x66, 0xdb, 0xa9, 0x4d, 0x8c, 0x44, 0x30, 0xd4, 0x0f, 0x72,
+	0x70, 0xf9, 0xe5, 0x3b, 0x4a, 0x2c, 0xad, 0xfe, 0x89, 0xe0, 0x63, 0x29, 0x2e, 0x39, 0xfd, 0xaa,
+	0xf1, 0x1a, 0xac, 0xe8, 0x96, 0xe5, 0x3c, 0xa4, 0x87, 0xc2, 0xfa, 0x52, 0x38, 0x58, 0xb5, 0xfc,
+	0x63, 0x12, 0xc0, 0xf1, 0xa7, 0xa0, 0xe8, 0x52, 0x9d, 0xc9, 0x8e, 0x5c, 0x0e, 0xeb, 0x8e, 0x88,
+	0x53, 0x22, 0xa1, 0xb8, 0x05, 0x55, 0x1a, 0x5f, 0x28, 0x4e, 0xda, 0x37, 0x92, 0xf8, 0xea, 0xbf,
+	0x10, 0xe0, 0x94, 0x3e, 0x65, 0xc4, 0xfa, 0x54, 0xeb, 0x6c, 0xcd, 0xe3, 0x7f, 0xa2, 0x47, 0xfd,
+	0x81, 0x37, 0xe5, 0xf4, 0x06, 0x15, 0x24, 0x25, 0x9a, 0x9b, 0x94, 0xe1, 0xfb, 0x6b, 0x6e, 0xee,
+	0xfb, 0x6b, 0x98, 0x8f, 0xf9, 0xf3, 0xc8, 0xc7, 0xdf, 0x21, 0x50, 0xe6, 0x19, 0x1d, 0xee, 0x72,
+	0xe8, 0xc5, 0xff, 0x1b, 0x95, 0x92, 0x64, 0xb9, 0x8c, 0x49, 0xf6, 0x5b, 0x04, 0xc9, 0xc9, 0x08,
+	0xd7, 0x83, 0xcd, 0x3b, 0xf2, 0x62, 0x23, 0x36, 0xef, 0x60, 0xe9, 0x5e, 0xc4, 0xe7, 0xe1, 0x9b,
+	0x77, 0xfe, 0x3c, 0xde, 0xbc, 0xdb, 0x5b, 0x4f, 0x9e, 0xd5, 0x96, 0x3e, 0x7c, 0x56, 0x5b, 0x7a,
+	0xfa, 0xac, 0xb6, 0xf4, 0x78, 0x52, 0x43, 0x4f, 0x26, 0x35, 0xf4, 0xe1, 0xa4, 0x86, 0x9e, 0x4e,
+	0x6a, 0xe8, 0x1f, 0x93, 0x1a, 0xfa, 0xe0, 0x79, 0x6d, 0xe9, 0xfd, 0xdc, 0x68, 0xfb, 0xbf, 0x01,
+	0x00, 0x00, 0xff, 0xff, 0x71, 0xf0, 0xd3, 0x3c, 0x02, 0x20, 0x00, 0x00,
```

Comment:
Not sure why I got this change... ran `make update-codegen-crds` with go 1.17.2.

Reply:
rolled back since the verify job broke